net/datacenter-node-install/setup-dc-node-2.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+if [[ "$EUID" -ne 0 ]]; then
+  echo "Please run as root"
+  exit 1
+fi
+
+HERE="$(dirname "$0")"
+
+set -xe
+
+"$HERE"/setup-cuda.sh
```

Comment:
Do we always want to run this?  Our CPU nodes don't need cuda.  Can we check for the presence of certain cards before running this?

To make things complicated, there ARE graphics cards in the CPU nodes, but cheapo ones for local console access only.

Reply:
My gut says, "it won't hurt," but I think this is another one we'd need an answer from @TristanDebrunner on.

We *could* parse the output of `lspci` to figure out if/what cards are installed